build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     }
     dependencies {
         classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.4.12'
-        classpath 'com.android.tools.build:gradle:3.0.0'
+        classpath 'com.android.tools.build:gradle:3.1.4'
```

Comment:
This might or might not cause other peoples Android Studios to complain and ask them to update their tools versions and/or Android Studio to complain. Not that its a big deal, but if we are aware of it we can preempt it.

Reply:
This is because Android Studio was complaining.